packages/slate-editor/src/extensions/image/components/ImageElement.tsx

```diff
@@ -88,6 +88,7 @@ export function ImageElement({
                 files: [initialFileInfo],
                 imagesOnly: true,
                 multiple: false,
+                tabs: [],
```

Comment:
Previously, when you clicked "Crop" in the image menu, it would spawn Uploadcare with the default tabs, so you could either crop the image or select a completely different one, which I think is really confusing behaviour.

Checked with Al and he agreed, so I've added the `tabs` property here which makes Uploadcare only show the cropping tool.